llvm/test/Analysis/LoopAccessAnalysis/forward-loop-carried.ll

```diff
@@ -62,10 +62,6 @@ define void @forward_different_access_sizes(ptr readnone %end, ptr %start) {
 ; CHECK-NEXT:      Report: unsafe dependent memory operations in loop. Use #pragma clang loop distribute(enable) to allow loop distribution to attempt to isolate the offending operations into a separate loop
 ; CHECK-NEXT:  Forward loop carried data dependence that prevents store-to-load forwarding.
 ; CHECK-NEXT:      Dependences:
-; CHECK-NEXT:        Forward:
```

Comment:
Should the dependence here be dropped? Is this due to the changes in how we get the size/scale as they should stride the same amount, but offset by sizeof(%struct)?